### DISCUSSION TITLE
Development build fails by missing package product 'ArgumentParser' in Xcode

### DISCUSSION THREAD

**Discussion by muukii** [swiftyfinch/Rugby#44] [Q&A] [answered]
I've tried to build Rugby as development in Xcode which is opened by `Package.swift`.
But I got an error in Swift packages that missing package product ArgumentParser.
Still building in command-line with `swift build` works well.
Anything should I do fix it?

Cloned from `main`
Opening with Xcode 12.5.1
M1 mac

<img width="782" alt="CleanShot 2021-08-24 at 15 31 15@2x" src="https://user-images.githubusercontent.com/1888355/130567611-311d1b01-a0fd-4db8-96cf-b66b3ed1456d.png">

**Comment by muukii** [comment, marked as answer]
Nah, cache was working bad. sorry. It works

**Reply by swiftyfinch** [reply]
Hello, did your Package file changed after resolving packages?
Probably, it can depends on moving `ArgumentParser` to Swift Package Collection 🤔
